rfcs/shared/build-system/03-packaging-for-npm.md

```diff
@@ -229,13 +230,14 @@ New build output structure will look like following:
 
 ```sh
 |- <package-name>/
+|-  dist/
+|-  |-  index.d.ts  # rolluped declaration file
```

Comment:
this setup is already in place and being published, so I decided to update to not introduce additional changes that dont provide much value (having `index.d.ts` in root )